src/binding/petsc4py/src/petsc4py/PETSc/DMDA.pyx

```diff
@@ -269,7 +552,19 @@ cdef class DMDA(DM):
 
     #
 
-    def getRanges(self):
+    # TODO: should we clarify how is getRanges different than getGhostRanges and getCorners?
```

Comment:
I don't think so. Just refer to the dmda manual section at the top of the class and refer to the specifi C api in each function

Reply:
maybe list all possible python getters in the see also